src/lazy/lazy_formatter.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+class LazyFormatter:
+	def __init__(self, valobj, dict):
+		self.valobj = valobj
+
+	def get_child_index(self, name):
+		return 0
+
+	def num_children(self):
+		if self.has_value:
+			return 1
+		return 0
+	
+	def get_child_at_index(self, index):
+		if index < 0:
+			return None
+		if not self.has_value:
+			return None
+		
+		value = (self.valobj
+			.GetChildMemberWithName('value_')
+			.GetChildMemberWithName('storage_')
+			.GetChildMemberWithName('value')
+		)
+
+		return value
+	
+	def update(self):
+		self.has_value = True if (self.valobj
+			.GetChildMemberWithName('value_')
+			.GetChildMemberWithName('storage_')
+			.GetChildMemberWithName('hasValue')
+			.GetValueAsUnsigned()
+		) == 1 else False
```

Comment:
) != 0 else False

this is safer because you can't always guarantee that a `true` value will be 1.